eacl/record.go

```diff
@@ -23,6 +23,32 @@ type Record struct {
 	targets   []Target
 }
 
+// CopyTo writes deep copy of the [Record] to dst.
+func (r Record) CopyTo(dst *Record) {
+	if dst == nil {
+		return
+	}
+
+	dst.action = r.action
+	dst.operation = r.operation
+
+	dst.filters = make([]Filter, 0, len(r.filters))
```

Comment:
length is known in advance, so we may avoid `newFilter` inside loop, same for target